installation/src/Model/DatabaseModel.php

```diff
@@ -380,6 +381,7 @@ public function createDatabase($options)
 					'password' => $options->db_pass_plain,
 					'prefix'   => $options->db_prefix,
 					'select'   => $options->db_select,
+					DatabaseHelper::getEncryptionSettings($options),
```

Comment:
Here the comma is ok because it is an array constructor and not another function call.